src/main/java/org/broadinstitute/consent/http/models/elastic_search/DatasetTerm.java

```diff
@@ -0,0 +1,170 @@
+package org.broadinstitute.consent.http.models.elastic_search;
+
+import java.util.List;
+import org.broadinstitute.consent.http.models.ontology.DataUseSummary;
+
+public class DatasetTerm {
+
+  private Integer datasetId;
+  private String datasetIdentifier;
+  private String description;
+  private String studyName;
+  private Integer studyId;
+  private Integer participantCount;
+  private String phenotype;
+  private String species;
+  private String piName;
+  private UserTerm dataSubmitter;
+  private String dataCustodian;
+  private DataUseSummary dataUse;
+  private List<String> dataTypes;
+  private String dataLocation;
+  private String dacName;
+  private Boolean openAccess;
+  private Boolean publicVisibility;
```

Comment:
I added public visibility because I felt it would be essential to know when displaying datasets to the user.